help/Gemfile.lock

```diff
@@ -0,0 +1,200 @@
+    jekyll-watch (2.2.1)
+      listen (~> 3.0)
+    kramdown (2.4.0)
+      rexml
+    kramdown-parser-gfm (1.1.0)
+      kramdown (~> 2.0)
+    liquid (4.0.4)
+    listen (3.9.0)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    mercenary (0.4.0)
+    mini_portile2 (2.8.7)
+    minima (2.5.2)
+      jekyll (>= 3.5, < 5.0)
+      jekyll-feed (~> 0.9)
+      jekyll-seo-tag (~> 2.1)
+    nokogiri (1.16.7)
+      mini_portile2 (~> 2.8.2)
+      racc (~> 1.4)
+    nokogiri (1.16.7-aarch64-linux)
+      racc (~> 1.4)
+    nokogiri (1.16.7-arm-linux)
+      racc (~> 1.4)
+    nokogiri (1.16.7-arm64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.16.7-x86-linux)
+      racc (~> 1.4)
+    nokogiri (1.16.7-x86_64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.16.7-x86_64-linux)
+      racc (~> 1.4)
+    pathutil (0.16.2)
+      forwardable-extended (~> 2.6)
+    public_suffix (6.0.1)
+    racc (1.8.1)
+    rake (13.2.1)
+    rb-fsevent (0.11.2)
+    rb-inotify (0.11.1)
+      ffi (~> 1.0)
+    rexml (3.3.7)
+    rouge (4.4.0)
+    safe_yaml (1.0.5)
+    sass-embedded (1.79.3)
+      google-protobuf (~> 4.27)
+      rake (>= 13)
+    sass-embedded (1.79.3-aarch64-linux-android)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-aarch64-linux-gnu)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-aarch64-linux-musl)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-aarch64-mingw-ucrt)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-arm-linux-androideabi)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-arm-linux-gnueabihf)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-arm-linux-musleabihf)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-arm64-darwin)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-riscv64-linux-android)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-riscv64-linux-gnu)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-riscv64-linux-musl)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86-cygwin)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86-linux-android)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86-linux-gnu)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86-linux-musl)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86-mingw-ucrt)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86_64-cygwin)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86_64-darwin)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86_64-linux-android)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86_64-linux-gnu)
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86_64-linux-musl)
+      google-protobuf (~> 4.27)
+    terminal-table (3.0.2)
+      unicode-display_width (>= 1.1.1, < 3)
+    unicode-display_width (2.6.0)
+    webrick (1.8.2)
+
+PLATFORMS
+  aarch64-linux
```

Comment:
I'm not sure why so many platforms here ... none of our other `Gemfile.locks` have so many platforms. Not sure if it matters though 🤷🏼